README.md

```diff
@@ -1863,7 +1863,7 @@ Other Style Guides
 ## Hoisting
 
   <a name="hoisting--about"></a><a name="14.1"></a>
-  - [14.1](#hoisting--about) `var` declarations get hoisted to the top of their closest enclosing function scope, their assignment does not. `const` and `let` declarations are blessed with a new concept called [Temporal Dead Zones (TDZ)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let#temporal_dead_zone_tdz). It’s important to know why [typeof is no longer safe](https://web.archive.org/web/20200121061528/http://es-discourse.com/t/why-typeof-is-no-longer-safe/15).
+  - [14.1](#hoisting--about) `var` declarations get hoisted to the top of their closest enclosing function scope, their assignment does not. `const` and `let` declarations are blessed with a new concept called [Temporal Dead Zones (TDZ)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let#temporal_dead_zone_tdz). It’s important to know why [typeof is no longer safe](https://web.archive.org/web/20200121061528/https://es-discourse.com/t/why-typeof-is-no-longer-safe/15).
```

Comment:
That site never had https, and the protocol in the archive.org query is irrelevant. I left this one as http for a reason.

Reply:
It have when we enter it with https i.e., [https](https://es.discourse.group/t/about-the-proposals-category/15) instead of [http](http://es.discourse.group/t/about-the-proposals-category/15), it redirects us to to same page.

Reply:
sure. but it's on HTTPS regardless, because of the https:// at the front.